packages/sandbox/index.ts

```diff
@@ -15,6 +15,6 @@ const provider = new RPCProvider(config);
   result.
 
 
-  console.log(result);
+    console.log(result);
```

Comment:
can we exclude this package from lint? it's just a sandbox for lib developers

Reply:
ingored sandbox from lint